tests/BulkTest.php

```diff
@@ -711,7 +711,7 @@ public function testSetShardTimeout(): void
     {
         $bulk = new Bulk($this->_getClient());
 
-        $this->assertSame($bulk, $bulk->setShardTimeout(10));
+        $this->assertSame($bulk, $bulk->setShardTimeout('10s'));
```

Comment:
`Bulk:: setShardTimeout()` expects an string and seems right:

https://www.elastic.co/guide/en/elasticsearch/reference/7.17/docs-bulk.html#docs-bulk-api-query-params

Reply:
good catch!